scripts/autotools/libmongoc/CheckSasl.m4

```diff
@@ -39,31 +45,25 @@ AS_IF([test "$PHP_MONGODB_SASL" = "cyrus" -o "$PHP_MONGODB_SASL" = "auto" -o "$P
                       $MONGODB_SHARED_LIBADD)
   fi
 
-  if test \( "$PHP_MONGODB_SASL" = "cyrus" -o "$PHP_MONGODB_SASL" = "yes" \) -a "$found_cyrus" != "yes"; then
+  if test "$PHP_MONGODB_SASL" = "cyrus" -a "$found_cyrus" != "yes"; then
     AC_MSG_ERROR([Cyrus SASL libraries and development headers could not be found])
   fi
 ])
 
-AS_IF([test "$PHP_MONGODB_SASL" = "auto"],[
+if test "$PHP_MONGODB_SASL" = "auto"; then
   PHP_MONGODB_SASL="no"
-])
-
-dnl Warn for unsupported values (e.g. Cyrus SASL search path)
-if test ! \( "$PHP_MONGODB_SASL" = "cyrus" -o "$PHP_MONGODB_SASL" = "no" \); then
-  AC_MSG_WARN([unsupported --with-mongodb-sasl value: $PHP_MONGODB_SASL])
 fi
 
 AC_MSG_CHECKING([which SASL library to use])
 AC_MSG_RESULT([$PHP_MONGODB_SASL])
 
-dnl Disable Windows SSPI and GSSAPI
+dnl Disable Windows SSPI
 AC_SUBST(MONGOC_ENABLE_SASL_SSPI, 0)
-AC_SUBST(MONGOC_ENABLE_SASL_GSSAPI, 0)
```

Comment:
This was removed in libmongoc a very long time ago ([CDRIVER-2654](https://jira.mongodb.org/browse/CDRIVER-2654)).